backend/drm/drm.c

```diff
@@ -208,10 +209,41 @@ static bool init_planes(struct wlr_drm_backend *drm) {
 		 * tracking of them for DRM lease clients.
 		 *
 		 * possible_crtcs is a bitmask of crtcs, where each bit is an
-		 * index into drmModeRes.crtcs. So if bit 0 is set (ffs starts
-		 * counting from 1), crtc 0 is possible.
-		 */
-		int crtc_bit = ffs(plane->possible_crtcs) - 1;
+		 * index into drmModeRes.crtcs. So if bit i is set crtc i is
+		 * possible.
+                 *
+		 * The loop below is selecting the first possible CRTC without a
+		 * associated plane of the considered type or, as a fallback,
+		 * the last possible CRTC.
+                 */
+
+		int crtc_bit ;
+		int num_crtcs = (int)drm->num_crtcs ;
+		assert(num_crtcs<=64);
```

Comment:
Style: add spaces around arithmetic operators

(Applies to the rest of the file as well)